### PR TITLE
Dependabot, images, README

### PR DESCRIPTION
**Pull request by lfrancke**
* Fixes dependabot hint
* Images were missing, I fixed it
* Updates README with a few notes and hint

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -12,4 +12,18 @@ To build the site install Antora and run:
...
+== Development
+
+During development, it can be helpful to change the playbook so that the source does point at the local directory instead of the remote git repository:
+
+[source,yaml]
+----
+content:
+  sources:
+  - url: ./
+    branches: main
```
Comment (siegfriedweber) [inline README.adoc:26]: Usually you do not develop on `main`, so it is better to reference the current `HEAD`:

```suggestion
    branches: HEAD
```

File: README.adoc [hunk 2]
```diff
@@ -12,4 +12,18 @@ To build the site install Antora and run:
 
     antora antora-playbook.yml
 
+WARNING: Due to a https://gitlab.com/antora/antora/-/issues/745[bug] this will _not_ currently work in Node 16. I recommend using the latest LTS release (14) until the issue is resolved.
+
+== Development
+
+During development, it can be helpful to change the playbook so that the source does point at the local directory instead of the remote git repository:
```
Comment (siegfriedweber) [inline README.adoc:19]: Another hint but probably not worth mentioning:

A git submodule does not count as a git repository.
```
$ antora antora-playbook.yml
error: Local content source must be a git repository
```

**Review by siegfriedweber — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (lfrancke) [reply README.adoc:19]: I added a NOTE, thank you!

**Review by siegfriedweber — APPROVED** [review]
LGTM

**Comment by lfrancke** [comment]
I had to resolve conflicts after the #62 merge - another review, sorry @siegfriedweber

**Review by siegfriedweber — APPROVED** [review]

**Pull request merged**